golden_response/staging/get_pop_obs/moscow.json

```diff
@@ -5,6 +5,286 @@
     "dc/p/c9yreypkne6wh": {
       "popType": "Person",
       "observations": [
+        {
+          "id": "dc/o/zxgjjf5jxdmp",
+          "type": "Observation",
+          "measuredProp": "count",
+          "measuredValue": 188700,
+          "measurementMethod": "WikidataPopulation",
+          "provenanceId": "dc/6p25cm",
+          "observationDate": "1785",
+          "statVarIds": ["TotalPopulation"]
```

Comment:
Do you know why Count_Person is not here?

Reply:
Now we have it - changed the stat var DCID in the test.